benchmark/src/main/java/com/datatorrent/benchmark/window/AbstractWindowedOperatorBenchmarkApp.java

```diff
@@ -121,6 +122,10 @@ protected O createWindowedOperator(Configuration conf)
     }
   }
 
+  protected void setUpdatedDataStorage(O windowedOperator, Configuration conf, SpillableComplexComponentImpl sccImpl)
+  {
```

Comment:
The AbstractWindowedOperatorBenchmarkApp was extended by KeyedWindowedOperatorBenchmarkApp and WindowedOperatorBenchmarkApp. Only KeyedWindowedOperatorBenchmarkApp need to setUpdatedDataStorage.